docs/source/conf.py

```diff
@@ -365,6 +368,23 @@ def setup(app):
     app.connect('autodoc-process-docstring', _autodoc_process_docstring)
     app.connect('build-finished', _build_finished)
 
+    chainercv_version = '0.10.0'
```

Comment:
How about drawing chainercv version from the installed one? (also document sources?)

Reply:
OK. I will change the code to use the version of ChainerCV that is installed.
Is it OK to hardcode version in `setup.py` for `docs` install mode?

Reply:
I think it is Okay with leaving a comment as `# see chainer/5434` .